docs: enhance profile README with badges, docs, and community links

mingcheng opened the pull request:
- Add more badges to the profile README
- Add official documentation link
- Add community and contributing guidelines
- Remove unnecessary logo
- Enhance the GitHub repository description

mingcheng committed:
docs: enhance profile README with badges, docs, and community links

- Add more badges to the profile README
- Add official documentation link
- Add community and contributing guidelines
- Remove unnecessary logo
- Enhance the GitHub repository description

Signed-off-by: mingcheng <mingcheng@apache.org>

chlins commented on profile/README.md:
```diff
@@ -7,10 +7,26 @@
 to be the best practice and standard solution in cloud-native architectures.
 It is hosted by the Cloud Native Computing Foundation(CNCF)as Incubating Level Project.
 
+### Repository
```
The repository should distinguish between what is a sub-component of a project and what is a sub-project, e.g. nydus should belong to is sub-project.

mingcheng replied:
updated

mingcheng committed:
fix: update profile README repository links

- Remove duplicate `nydus`, `client`, and `api` entries
- Consolidate repository listings for clarity

Signed-off-by: mingcheng <mingcheng@apache.org>
